estela-api/api/views/stats.py

```diff
@@ -126,51 +142,51 @@ def summarize_stats_results(
                 stats_mapping["scraped_pages"], 0
             )
             stats_results[date_str]["pages"]["missed_pages"] += stats.get(
-                stats_mapping["received_count"], 0
+                stats_mapping["total_pages"], 0
             ) - stats.get(stats_mapping["scraped_pages"], 0)
             stats_results[date_str]["pages"]["total_pages"] += stats.get(
-                stats_mapping["received_count"], 0
+                stats_mapping["total_pages"], 0
             )
 
             stats_results[date_str]["status_codes"]["status_200"] += stats.get(
-                "downloader/response_status_count/200", 0
+                stats_mapping["status_200"], 0
             )
             stats_results[date_str]["status_codes"]["status_301"] += stats.get(
-                "downloader/response_status_count/301", 0
+                stats_mapping["status_301"], 0
             )
             stats_results[date_str]["status_codes"]["status_302"] += stats.get(
-                "downloader/response_status_count/302", 0
+                stats_mapping["status_302"], 0
             )
             stats_results[date_str]["status_codes"]["status_401"] += stats.get(
-                "downloader/response_status_count/401", 0
+                stats_mapping["status_401"], 0
             )
             stats_results[date_str]["status_codes"]["status_403"] += stats.get(
-                "downloader/response_status_count/403", 0
+                stats_mapping["status_403"], 0
             )
             stats_results[date_str]["status_codes"]["status_404"] += stats.get(
-                "downloader/response_status_count/404", 0
+                stats_mapping["status_404"], 0
             )
             stats_results[date_str]["status_codes"]["status_429"] += stats.get(
-                "downloader/response_status_count/429", 0
+                stats_mapping["status_429"], 0
             )
             stats_results[date_str]["status_codes"]["status_500"] += stats.get(
-                "downloader/response_status_count/500", 0
+                stats_mapping["status_500"], 0
             )
```

Comment:
Maybe this can be refactored.

Reply:
@webtaken I agree with @Adriana618 . Please refactor this part. We can surely use a `for` to iterate through the statuses. The same with the logs stats.